config.json

```diff
@@ -2464,6 +2464,26 @@
           "multiple-clause-matching"
         ],
         "difficulty": 5
+      },
+      {
+        "slug": "affine-cipher",
+        "name": "Affine Cipher",
+        "uuid": "18d12bac-ead0-456c-bcef-ccb2fd06fe72",
+        "prerequisites": [
+          "pattern-matching",
+          "atoms",
+          "pipe-operator",
+          "enum",
+          "integers",
+          "if",
+          "strings",
+          "tuples"
+        ],
+        "practices": [
+          "pipe-operator",
+          "strings"
+        ],
```

Comment:
There is a lot going on in this exercise, so I find it really hard to say what "practices" would be best. I think those make sense.